indico/modules/categories/controllers/management.py

```diff
@@ -185,27 +208,25 @@ def _set_image(self, data, metadata):
 
 class RHManageCategoryProtection(RHManageCategoryBase):
     def _process(self):
-        form = CategoryProtectionForm(obj=self._get_defaults(), category=self.category)
+        category = self.category
```

Comment:
Why?

Reply:
Good point indeed, not sure why we did it like this for event protection.... maybe the longer form with `self` required some wrapping below that made it look worse? But now it should be fine so maybe update it here and in `RHEventProtection`